notebooks/CubevizExample.ipynb

```diff
@@ -54,7 +50,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "viz.show_in_sidecar()"
+    "viz.show(loc='sidecar')"
```

Comment:
I really don't like these being run-able cells in example notebooks. They are already documented in `display.rst`. If you must keep them, please change them into "raw nbconvert" cell type, so people don't accidentally run all the ways to display because they are not supposed to.

Reply:
Removed these cells in all the notebooks!